pkg/workerpool/pool_test.go

```diff
@@ -55,7 +54,10 @@ func (s *workerPoolSuite) TestTaskError(c *check.C) {
 		c.Assert(err, check.ErrorMatches, "test error")
 	})
 
+	var wg sync.WaitGroup
+	wg.Add(1)
 	errg.Go(func() error {
```

Comment:
`errg.Go` is not necessary for this goroutine

Reply:
Thanks for your review!  💚 💙 💜 💛 ❤️